.github/workflows/ci.yml

```diff
@@ -138,9 +138,13 @@ jobs:
 
       - name: cargo clippy
         run: cargo hack clippy --workspace --locked --target wasm32-unknown-unknown --optional-deps --each-feature -- -D warnings
+        env:
+          RUSTFLAGS: '--cfg getrandom_backend="wasm_js"'
 
       - name: cargo clippy (auxiliary)
         run: cargo hack clippy --workspace --locked --target wasm32-unknown-unknown --optional-deps --each-feature --tests --benches --examples -- -D warnings
+        env:
+          RUSTFLAGS: '--cfg getrandom_backend="wasm_js"'
```

Comment:
In https://github.com/linebender/vello/pull/807, we suggest adding this to `.cargo/config.toml`. I think we should only use one of those